try{ 
	core.init();
} catch(e){
	console.log(e);
}